internal/controller/s3utils_test.go

```diff
@@ -83,9 +85,13 @@ type fakeObjectStorer struct {
 	name       string
 	bucketName string
 	objects    map[string]interface{}
+	mutex         sync.Mutex
```

Comment:
Formatting is off, looks like your editor is not configured properly. It format the file with `gofmt` or `golangci-lint fmt` when saving.